src/actions.js

```diff
@@ -8,10 +8,41 @@ export const initGrid = (width, height) => {
 };
 
 export const TURN_BLUE = "TURN_BLUE";
-export const turnBlue = (x, y) => {
+export const turnBlue = (x, y, clickedNum) => {
```

Comment:
I see. Was wondering about that. I had it set it up before I had the prop built in from the reducer